src/web100-util.c

```diff
@@ -688,7 +688,7 @@ int tcp_stat_get_data(tcp_stat_snap* snap, int testsock, int ctlsock,
     }
     snprintf(line, sizeof(line), "%s: %s\n",
                  estats_var_array[j].name, str);
-    send_json_message(ctlsock, TEST_MSG, line, strlen(line), testoptions->connection_flags, JSON_SINGLE_VALUE);
+    send_json_message_any(ctl, TEST_MSG, line, strlen(line), jsonSupport, JSON_SINGLE_VALUE);
```

Comment:
I'm really confused. Where is `jsonSupport` coming from? I don't see it defined in this file or in any header.